package-scripts.js

```diff
@@ -276,7 +276,7 @@ module.exports = {
       },
       postbuild: {
         script:
-          'buildProduction docs/_site/index.html --outroot docs/_dist --canonicalroot https://mochajs.org/ --optimizeimages --svgo --inlinehtmlimage 9400 --inlinehtmlscript 0 --asyncscripts && cp docs/_headers docs/_dist/_headers && node scripts/netlify-headers.js >> docs/_dist/_headers',
+          'buildProduction docs/_site/index.html --outroot docs/_dist --canonicalroot https://mochajs.org/ --optimizeimages --svgo --inlinehtmlimage 0 --inlinehtmlscript 0 --inlinecssimage 0 --asyncscripts && cp docs/_headers docs/_dist/_headers && node scripts/netlify-headers.js >> docs/_dist/_headers',
```

Comment:
I think we can adjust some of the inlining parameters, but then I also think we need to explicitly inline the logo, seeing as this change caused a regression in hero rendering and lost us the animation of the logo: http://webpagetest.org/video/compare.php?tests=190303_XT_8d430aac89f487867e1c34a2091db20c-r:1-c:0,190303_R9_221121187cf891367ca68538dc27e542-r:1-c:0